src/openzeppelin/token/erc1155/presets/ERC1155MintableBurnable.cairo

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.5.1 (token/erc1155/presets/ERC1155MintableBurnable.cairo)
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+from openzeppelin.access.ownable.library import Ownable
+from openzeppelin.token.erc1155.library import ERC1155
+from openzeppelin.introspection.erc165.library import ERC165
+
+//
+// Constructor
+//
+
+@constructor
+func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    uri: felt, owner: felt
+) {
+    ERC1155.initializer(uri);
+    Ownable.initializer(owner);
+    return ();
+}
+
+//
+// Getters
+//
+
+@view
+func supportsInterface{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    interfaceId: felt
+) -> (success: felt) {
+    return ERC165.supports_interface(interfaceId);
+}
+
+@view
+func uri{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(id: Uint256) -> (
+    uri: felt
+) {
+    return ERC1155.uri(id);
+}
+
+@view
+func balanceOf{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    account: felt, id: Uint256
+) -> (balance: Uint256) {
+    return ERC1155.balance_of(account, id);
+}
+
+@view
+func balanceOfBatch{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    accounts_len: felt, accounts: felt*, ids_len: felt, ids: Uint256*
+) -> (balances_len: felt, balances: Uint256*) {
+    let (balances_len, balances) = ERC1155.balance_of_batch(accounts_len, accounts, ids_len, ids);
+    return (balances_len, balances);
+}
```

Comment:
```suggestion
) -> (balances_len: felt, balances: Uint256*) {
    return ERC1155.balance_of_batch(accounts_len, accounts, ids_len, ids);
}
```
This only works if we rename the `balance_of_batch` method return values in the library. I think that can be done without any inconvenience.

Reply:
the priority is a clean and easy to use API, so i moved the burden to the lib. this will probably change soon with 1.0 though